database/seed.js

```diff
@@ -0,0 +1,24 @@
+const db = require('./index.js');
+
+const seed = () => {
+  return db.navbarSave(seedGen());
+}
+
+seedGen = function () {
+  var bundleId, countDown;
+  var timerData = [];
+  for (let i = 1; i < 100; i++) {
+    bundleId = i;
+    timer = Number(Math.floor (Math.random() * 13) + 1);
```

Comment:
Same consistency point here as in the index.js file in this directory just between `Number(...)` and `Math.floor (...)`.

Reply:
Of course on my screen the line break in the comment renders it useless, but note the space on line 12 after `Math.floor` is not present after the `Number` and `Math.random` functions are invoked.